packages/app/src/app/overmind/effects/gql/dashboard/mutations.ts

```diff
@@ -1,27 +1,273 @@
-// import {
-//   CreateCommentMutation,
-//   CreateCommentMutationVariables,
-//   DeleteCommentMutation,
-//   DeleteCommentMutationVariables,
-//   ResolveCommentMutation,
-//   ResolveCommentMutationVariables,
-//   UnresolveCommentMutation,
-//   UnresolveCommentMutationVariables,
-//   UpdateCommentMutation,
-//   UpdateCommentMutationVariables,
-// } from 'app/graphql/types';
-// import gql from 'graphql-tag';
-// import { Query } from 'overmind-graphql';
-
-// import { commentFragment } from './fragments';
-
-// export const unresolveComment: Query<
-//   UnresolveCommentMutation,
-//   UnresolveCommentMutationVariables
-// > = gql`
-//   mutation UnresolveComment($commentId: ID!, $sandboxId: ID!) {
-//     unresolveComment(commentId: $commentId, sandboxId: $sandboxId) {
-//       id
-//     }
-//   }
-// `;
+import {
+  UnmakeSandboxesTemplateMutation,
+  UnmakeSandboxesTemplateMutationVariables,
+  CreateCommentMutation,
+  CreateCollectionMutationVariables,
+  RenameFolderMutation,
+  DeleteFolderMutation,
+  DeleteFolderMutationVariables,
+  RenameFolderMutationVariables,
+  AddToFolderMutation,
+  AddToFolderMutationVariables,
+  MoveToTrashMutation,
+  MoveToTrashMutationVariables,
+  ChangePrivacyMutationVariables,
+  _PermanentlyDeleteSandboxesMutationVariables,
+  ChangePrivacyMutation,
+  _PermanentlyDeleteSandboxesMutation,
+  _LeaveTeamMutationVariables,
+  _LeaveTeamMutation,
+  _RemoveFromTeamMutation,
+  _RemoveFromTeamMutationVariables,
+  _InviteToTeamMutation,
+  _InviteToTeamMutationVariables,
+  _RevokeTeamInvitationMutation,
+  _RevokeTeamInvitationMutationVariables,
+  _AcceptTeamInvitationMutation,
+  _AcceptTeamInvitationMutationVariables,
+  _RejectTeamInvitationMutation,
+  _RejectTeamInvitationMutationVariables,
+  _SetTeamDescriptionMutation,
+  _SetTeamDescriptionMutationVariables,
+  _CreateTeamMutation,
+  _CreateTeamMutationVariables,
+  _RenameSandboxMutation,
+  _RenameSandboxMutationVariables,
+  MakeSandboxesTemplateMutation,
+  MakeSandboxesTemplateMutationVariables,
+} from 'app/graphql/types';
+import gql from 'graphql-tag';
+import { Query } from 'overmind-graphql';
+
+import {
+  teamFragmentDashboard,
+  sidebarCollectionDashboard,
+  sandboxFragmentDashboard,
+} from './fragments';
+
+export const createTeam: Query<
+  _CreateTeamMutation,
```

Comment:
All the underscores are because two queries/mutations cannot have the same name and these already exist in the "old" dashboard